apps/settings/serializers/public.py

```diff
@@ -34,6 +34,8 @@ class PrivateSettingSerializer(PublicSettingSerializer):
     PASSWORD_RULE = serializers.DictField()
     SECURITY_SESSION_SHARE = serializers.BooleanField()
     XPACK_LICENSE_IS_VALID = serializers.BooleanField()
+    XPACK_LICENSE_EDITION_ULTIMATE = serializers.BooleanField()
+    FACE_RECOGNITION_ENABLED = serializers.BooleanField()
     XPACK_LICENSE_INFO = serializers.DictField()
     HELP_DOCUMENT_URL = serializers.CharField()
     HELP_SUPPORT_URL = serializers.CharField()
```

Comment:
I'm afraid your request is a bit unclear. You've mentioned differences between code fragments (from 2021) and you'd like to know if there's anything irregular or problematic about them, whether those problems have an impact on functionality, or how to improve it? Could you please provide more context or clarify this request?